php/lang/security/injection/echoed-request.php

```diff
@@ -44,6 +44,22 @@ function doSmth5() {
     echo "Hello ".trim($_POST['name']);
 }
 
+function doSmth6() {
+   $VAR = $_GET['someval']
```

Comment:
typo, missing semicolon - missed this a couple times, causing a parse error 
```suggestion
   $VAR = $_GET['someval'];
```